scripts/2022_05_26_add_frax3pool_to_minichef.py

```diff
@@ -0,0 +1,52 @@
+from helpers import CHAIN_IDS, MINICHEF_ADDRESSES, MULTISIG_ADDRESSES, SDL_ADDRESSES
+from ape_safe import ApeSafe
+from brownie import accounts, network
+
+from scripts.utils import confirm_posting_transaction
+
+
+def main():
+    """Add SimpleRewarder for T to minichef"""
+
+    print(f"You are using the '{network.show_active()}' network")
+    deployer = accounts.load("deployer")  # prompts for password
+    multisig = ApeSafe(MULTISIG_ADDRESSES[CHAIN_IDS["MAINNET"]])
+
+    minichef = multisig.contract(MINICHEF_ADDRESSES[CHAIN_IDS["MAINNET"]])
+    poolLength = minichef.poolLength()
+    newPoolIndex = poolLength + 1
+    minichef.massUpdatePools(["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"])
+
+    # Static
+    currentSDLPerSecond = minichef.saddlePerSecond()
+    frax3poolSDLPerSecond = (2_000_000 * 1e18) / (2 * 4 * 7 * 24 * 60 * 60)
+    totalAlloc = minichef.totalAllocPoint()
+
+    # Calculation
+    newSDLPerSecond = currentSDLPerSecond + frax3poolSDLPerSecond
+    fraxAllocationRate = frax3poolSDLPerSecond / newSDLPerSecond
+    fraxAlloc = round((totalAlloc * fraxAllocationRate) / (1 - fraxAllocationRate))
```

Comment:
I checked this math by confirming that `fraxAlloc / (totalAlloc + fraxAlloc) = fraxAllocationRate`

Reply:
was changed to `pids = [x for x in range(1, pool_length)]`